Exiled.Events/EventArgs/ExplodingGrenadeEventArgs.cs

```diff
@@ -81,8 +92,14 @@ public ExplodingGrenadeEventArgs(Player thrower, EffectGrenade grenade, List<Pla
         /// <summary>
         /// Gets a value indicating whether the grenade is a frag or flash grenade.
         /// </summary>
+        [Obsolete("Use GrenadeType instead.")]
```

Comment:
[Obsolete("Use GrenadeType instead.", true)] so that it causes a compilation error, to keep tinybrain people who ignore warnings from not realizing it's replaced